### DISCUSSION TITLE
Storing data inside rooms

### DISCUSSION THREAD

**Discussion by Taykavol** [socketio/socket.io#4197] [Q&A]
Hey, I am developing chess game as hobby and face with problem. We have an option to store data via socket object socket.data, but how to store data inside room, I need to keep a chess instanse during the whole game. I didn't find this feature.  I think it's very usefull feature. It's very important to keep scalability between server instances (must be working well with adapters).

**Comment by darrachequesne** [comment]
This feature does not exist currently, but I think it might make sense to add it. Do you have a particular API in mind?

Note: actually it would make the room behave like an actual room, and less like a subscription
